tests/foreman/cli/test_errata.py

```diff
@@ -459,6 +466,149 @@ def test_negative_install_by_hc_name_without_org_info(self):
             )
         self.assertIn('Error: Could not find organization', context.exception.stderr)
 
+    @tier3
+    def test_positive_list_affected_chosts_by_erratum_restrict_flag(self):
+        """View a list of affected content hosts for an erratum filtered
+        with restrict flags. Applicability is calculated using the Library,
+        so that search must not limit to CV or LCE. Installability
+        is calculated using the attached CV, subject to the CV's own filtering,
+        so that search must limit to CV and LCE.
+
+        :id: 594acd48-892c-499e-b0cb-6506cea7cd64
+
+        :Setup: Errata synced on satellite server.
+
+        :Steps:
+
+            1. erratum list --erratum-id=<erratum_id>
+               --organization-id=<org_id> --errata-restrict-installable=1
+
+            2. erratum list --erratum-id=<erratum_id>
+               --organization-id=<org_id> --errata-restrict-installable=0
+
+            3. erratum list --erratum-id=<erratum_id>
+               --organization-id=<org_id> --errata-restrict-applicable=1
+
+            4. erratum list --erratum-id=<erratum_id>
+               --organization-id=<org_id> --errata-restrict-applicable=0
+
+
+        :expectedresults: List of affected content hosts for an erratum is
+            displayed filtered with corresponding restrict flags.
+
+        :CaseAutomation: automated
+
+        """
+        # Create list of uninstallabe errata by removing FAKE_2_ERRATA_ID
+        UNINSTAllABLE = FAKE_9_YUM_ERRATUM[1:5]
```

Comment:
Can we make that more explicit and not depend of order of items in constant?
```python
UNINSTAllABLE = [erratum for erratum in FAKE_9_YUM_ERRATUM if erratum != FAKE_2_ERRATA_ID]
```
Or use set operations, but mind that will shuffle order of items inside. I think it doesn't matter in this case, but I'm adding that to let you know.
```python
UNINSTAllABLE = set(FAKE_9_YUM_ERRATUM) - set(FAKE_2_ERRATA_ID)
# just to let you know, you can wrap right side in `list()` if something down the line requires list and not any iterable; in this case, you don't need this
UNINSTAllABLE = list(set(FAKE_9_YUM_ERRATUM) - set(FAKE_2_ERRATA_ID))
```
Also, why double-L is lowercase in variable name?

Reply:
Hi @mirekdlugosz , I will go with the first idea. Thank you.